[FIX] Index admin-created proposals.

tramuntanal opened the pull request:
#### :tophat: What? Why?


#### :pushpin: Related Issues
- Fixes #4540 

#### :clipboard: Subtasks
- [x] Add `CHANGELOG` entry
- [-] Add documentation regarding the feature 
- [-] Add/modify seeds
- [x] Add tests
- [x] Do the fix.

mrcasals commented:
@tramuntanal tests fail because Rubocop complains! Also, there's a conflict in the CHANGELOG 😄

tramuntanal commented:
Master is unstable, see you on monday :D

mrcasals commented:
While it's true `master` is failing, the errors in your PR are not the same we're encountering in `master`. The errors you're encountering are derived from this PR code.

tramuntanal commented:
But the failures and modules failing were not related with what I changed I think to remember..
Well, I've just updated from master and will check again more carefully

mrcasals commented:
Failures in this PR happened during the seed phase (both generators and core), master was failing due to a timeout (see #4608).

As yu can see, this PR keeps failing with the same error 😅

tramuntanal commented:
Yes! you are right, I see the problem arises when indexing seeds data. I'll check. Thanks!
